### ISSUE TITLE
Remove -moz-border-radius and -webkit-border-radius from core files

### ISSUE THREAD

**Issue by izmeez** [backdrop/backdrop-issues#6265]
## Description of the need

The use of `-moz-border-radius` and `-webkit-border-radius` generate numerous errors visible in developer tools as `Unknown property`. They have been removed from newer versions of Firefox and Chrome browsers respectively. 

Removing these from core files will reduce excessive error messages in developer tools.

Is it time to remove them from core files or is it necessary to keep them to support older browsers?

## Additional information

This relates to 18 files in core:
```
core/misc/dropbutton.css
core/modules/contextual/css/contextual.css
core/modules/node/css/node.preview.css
core/modules/simpletest/files/css_test_files/css_input_without_import.css
core/modules/simpletest/files/css_test_files/css_input_without_import.css.optimized.css
core/modules/simpletest/files/css_test_files/css_input_without_import.css.unoptimized.css
core/modules/system/css/menudropdown.theme.css
core/modules/system/css/menudropdown.theme.breakpoint-queries.css
core/modules/system/css/system.theme.css
core/modules/views_ui/css/views_ui.admin.theme.css
core/themes/bartik/color/preview.css
core/themes/bartik/css/editor.css
core/themes/bartik/css/style.css
core/themes/basis/css/component/admin-tabs.css
core/themes/seven/css/jquery.ui.theme.css
core/themes/seven/css/responsive-tabs.css
core/themes/seven/css/style.css
core/themes/seven/css/views-admin.seven.css
```

The `-moz-border-radius` styling was dropped from the Firefox browser a long time ago, as was `-webkit-border-radius` from the Chrome browser. The latter is present in 13 core files.

These can be replaced with the `border-radius` styling.

Should these be changed or kept to support old browsers?

**Comment by klonos** [comment]
Thanks for taking the time to start this issue @izmeez 🙏🏼 

We should clean up our CSS from such deprecated vendor-specific properties if they have been standardized for years.

Having said that, besides older/unsupported versions of IE (#214 anyone?), I don't think that we have a specific policy around supported browsers more generally (but we should). There's only https://docs.backdropcms.org/documentation/system-requirements which is about the server-side software required to run a backdrop site - not what's required/recommended in order to browse it.

Related: https://www.drupal.org/docs/system-requirements/browser-requirements

**Comment by quicksketch** [comment]
I filed a PR at https://github.com/backdrop/backdrop/pull/4570 that removes not only `border-radius` but also `box-sizing` and `box-shadow` vendor prefixes, which have been standardized for just as long as border-radius. Other vendor prefixes like gradients, flexbox support, transform, and text selection I left in place.

**Comment by quicksketch** [comment]
@albanycomputers reviewed the PR at https://github.com/backdrop/backdrop/pull/4570 (thank you!) and pointed out there are still several other prefixes in use. Of those, I think that `transition` could have its prefixes removed safely from all browsers, but I'm not 100% sure. I'd like to start with these obvious and oldest ones first, then we could consider removing other prefixes that apply only to some browsers.

**Comment by laryn** [comment]
I agree on getting this included in 1.27.0 and on starting with the oldest/obvious ones. I think transition is very safe.

 - https://caniuse.com/?search=transition

**Comment by quicksketch** [comment]
Thanks @laryn. I went ahead and made an additional commit that removes `-webkit` and other prefixes for `transition`, `transform`, `animation`, and backgrounds using `linear-gradient`. All 4 features seam to be universally available from 2014 forward (IE9 era).

**Comment by indigoxela** [comment]
This looks great, one minor spacing issue, everything else is ready to go.

**Comment by laryn** [comment]
Thanks @quicksketch for the PR, and @izmeez, @indigoxela, @albanycomputers, and @klonos for review and input. Merged and will be included in `1.27.0`.
